### DISCUSSION TITLE
Can you use svgs as custom nodes

### DISCUSSION THREAD

**Discussion by slechanii** [xyflow/xyflow#1399] [Q&A]
Hi, i'm currently using react-d3-graph and consider switching to this library instead.

One of the features that I really like in the other library is that you can customize your nodes using different svgs together, i saw that it was possible to customize nodes in this library (color picker example) but didn't see any svg being used, can someone confirm me that you can easily customize your nodes using different svgs to build them before I start switching please ?

Have a good day

**Comment by Azim-Ahmed** [comment]
Yes you can, You just need to work with custom svg file in custom node components. I have just tried after your discussion comment. And it works

**Reply by marcel-burda** [reply]
Is it possible to give a minimal example to that? I could not figure out how to pass my svg to the costum node components.

Cheers

**Reply by nadiTime** [reply]
@marcel-burda did you manage to implement node shape with svg?
